src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -2273,6 +2275,8 @@ gregtech.machine.quantum_chest.iv.name=Black Hole-Quantum Chest III
 #Quantum tanks
 gregtech.machine.quantum_tank.tooltip=Compact place to store all your fluids.
 gregtech.machine.quantum_tank.capacity=Fluid Capacity: %,d mb
+gregtech.machine.quantum_tank.tooltip.name=Fluid Stored: §b%s
+gregtech.machine.quantum_tank.tooltip.count=Fluid Amount: §e%,d
```

Comment:
This localization should have `mb` after the number, so the tooltip would look like `64,000,000 mb` rather than `64,000,000`